Makefile

```diff
@@ -21,6 +21,10 @@ setup-dev: pipenv
 	$(PIPENV) install --verbose --dev --python $(PYTHON_VERSION)
 	$(PIPENV_PIP) install deps/h2o_wave-nightly-py3-none-manylinux1_x86_64.whl --force-reinstall
 
+.PHONY: export-requirements
+export-requirements: pipenv
+	$(PIPENV) requirements > requirements.txt
```

Comment:
Can we somehow add the nightly wave build there?

Reply:
Only very manual, as pipenv can't handle the non-pep8 version.

Reply:
We can add it to the requirements, but not automatically with pipenv.

Reply:
Added it manually to `makefile`.